docs/testing.md

```diff
@@ -1,5 +1,13 @@
 # Writing Tests
 
+## Controlling test verbosity
+
+To run tests verbosely (displaying test events and output and error streams to the console), use the following system property:
+
+```shell
+./gradlew test -Dorg.gradle.project.verboseTest
```

Comment:
You can set the project property more concisely using: `./gradlew test -PverboseTest`